src/eko/couplings.py

```diff
@@ -123,6 +127,8 @@ def expanded_nnlo(ref, beta0, beta1, beta2, lmu):
 def expanded_n3lo(ref, beta0, beta1, beta2, beta3, lmu):
     """Compute expanded solution at N3LO.
 
+    From Luca Rottoli
```

Comment:
- maybe we should add a bibtex entry á là "Luca Rottoli, private communication" and cite that one here
- also mention that is obtained via ordinary iterated solution of the RGE